vitrina/templates/component/navbar.html

```diff
@@ -49,8 +49,8 @@
                             <div class="navbar-dropdown">
                                 <a href="/user/profile" class="navbar-item">{% translate "auth.profile" %}</a>
                                 <a href="{% url 'user-task-list' user.pk %}" class="navbar-item">{% translate "Užduotys" %}</a>
-                                {% if user.role is not empty %}
-                                    <a href="/sso/admin" class="navbar-item">Administruoti</a>
+                                {% if user.is_authenticated and user.is_staff %}
+                                    <a href="/admin/" class="navbar-item">{% translate "Administruoti" %}</a>
```

Comment:
Do not hardcode URLs. Reverse `admin:index`.

https://docs.djangoproject.com/en/3.2/ref/contrib/admin/#reversing-admin-urls